frontend/src/services/objectService.ts

```diff
@@ -227,7 +228,54 @@ export default {
   searchObjects(params: SearchObjectsOptions = {}, headers: any = {},) {
     // remove objectId array if its first element is undefined
     if (params.objectId && params.objectId[0] === undefined) delete params.objectId;
-    return comsAxios().get(PATH, { params: params, headers: headers });
+
+    if (params.objectId) {
+      /**
+       * split calls to COMS if query params (eg objectId's)
+       * will cause url length to excede 2000 characters
+       * see: https://stackoverflow.com/questions/417142/what-is-the-maximum-length-of-a-url-in-different-browsers
+       *
+       * TODO: consider creating a utils function
+       * eg: `divideParam(params, attr)`
+       *     ... return Promise.all(divideParam(params, objectId).map(zparam => comsAxios().get(PATH, {params: zparam, headers: headers});
+       */
+      let urlLimit = 2000;
+      // minus base url
+      const baseUrl = new URL(`${new ConfigService().getConfig().coms.apiPath}${PATH}`).toString().length;
+      urlLimit = urlLimit - baseUrl;
+      // minus deleteMarker=false
+      urlLimit = urlLimit - 19;
+      // minus latest=false
+      urlLimit = urlLimit - 13;
+      // minus a single bucketId (bucketId[]=<uuid>)
+      if (params.bucketId) { urlLimit = urlLimit - 48; }
+      // if tagset parameters passed
+      if (params.tagset) {
+        for (const [key, value] of Object.entries(params.tagset)) {
+          // @ts-ignore
+          urlLimit = urlLimit - 10 - key.length - value.length;
```

Comment:
I'm assuming you're encountering some shenanigans with typescript to just put in a blanket ignore? We should try to resolve the problem instead of just ignoring it, even if it is a bit pedantic. In the event it's the lack of definition for `Object.prototype.entries`, we could try this typing:
```ts
for (const [key, value] of Object.entries<string>(params.tagset): {[K in keyof T]: [K, T[K]]}[keyof T][]) {
  urlLimit -= 10 + key.length + value.length; // subtract k/v pair and boilerplate
}
```
Some inspiration from here: <https://github.com/microsoft/TypeScript/issues/35101>
If it doesn't work, it's still better we force-cast it as an `any` type instead of ignoring it if possible.